docs/workflows/ci_cd_updated_default_branch.md

```diff
@@ -46,6 +46,7 @@ Inputs related to updating the permanent dependencies branch.
 
 | **Name** | **Description** | **Required** | **Default** | **Type** |
 |:--- |:--- |:---:|:---:|:---:|
+| `update_dependencies_branch` | Whether or not to update the permanent dependencies branch. | No | `true` | _boolean_ |
 | `permanent_dependencies_branch` | The branch name for the permanent dependency updates branch. | No | ci/dependency-updates | _string_ |
```

Comment:
Consider adding a deprecation warning here as well

Reply:
Sorry. I don't understand anything of this comment?

The input `permanent_dependencies_branch` is used in multiple callable workflows. The one you've tagged here is the documentation page for the callable workflow `ci_cd_updated_default_branch`, which I haven't deprecated the `permanent_dependencies_branch` input for, nor added a `target_branch` for. Rather, this has only been done for the callable workflow `ci_check_pyproject_dependencies`. The fact that the input variable `permanent_dependencies_branch` is called the same in the two callable workflows is not relevant.